packages/components/addon/components/hds/code-block/index.hbs

```diff
@@ -0,0 +1,30 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+
+<div class="hds-code-block-wrapper">
+  <div
+    class={{this.classNames}}
+    {{style maxHeight=@maxHeight}}
+    {{hds-code-mirror
+      value=@value
+      options=@options
+      lineNumbers=@lineNumbers
+      lineWrapping=@lineWrapping
+      readOnly=@readOnly
```

Comment:
probably the `readonly` use case is the most common one (so should be `true` by default)?

Reply:
Probably so, I can change it.